src/envoy/http/authn/filter_context.cc

```diff
@@ -71,6 +72,35 @@ void FilterContext::setPrincipal(const iaapi::PrincipalBinding& binding) {
   }
 }
 
+bool FilterContext::getJwtPayload(const std::string& issuer,
+                                  std::string* payload) const {
+  // auto location_iter =
+  //     filter_config_.jwt_output_payload_locations().find(issuer);
+  // if (location_iter == filter_config_.jwt_output_payload_locations().end()) {
+  //   ENVOY_LOG(warn, "No JWT payload output location is found for the issuer
+  //   {}", issuer); return false;
+  // }
+  const auto filter_it =
+      dynamic_metadata_.filter_metadata().find(istio::utils::FilterName::kJwt);
+  if (filter_it == dynamic_metadata_.filter_metadata().end()) {
```

Comment:
Is this using the stuff that went into envoy recently (for exchanging info between filters)? If so, doesn't that require filters to declare dependencies ?

Reply:
By dependencies do you mean having the right order in filter chain? Pilot code (i.e plugin) is responsible to do that properly. It's already the existing behavior.